WalletWasabi/CoinJoin/Coordinator/Rounds/CoordinatorRound.cs

```diff
@@ -1199,11 +1214,16 @@ public async Task<IEnumerable<Alice>> RemoveAlicesIfAnInputRefusedByMempoolAsync
 						Logger.LogInfo($"Mempool acceptance failed: {result.reason ?? "no reason"}.");
```

Comment:
Why not log the removed Alice directly after this, so we can remove the next foreach.

Reply:
I am not sure. Probably better to log in this way for some reason. Changing this would change the order of the log entries.